editor.planx.uk/src/@planx/graph/index.ts

```diff
@@ -46,7 +47,7 @@ const isExternalPortalNodeType = (id: string, graph: Graph): boolean =>
 
 const sanitize = (x: any) => {
   if ((x && typeof x === "string") || x instanceof String) {
-    return trim(x.replace(/[\u200B-\u200D\uFEFF↵]/g, ""));
+    return DOMPurify.sanitize(trim(x.replace(/[\u200B-\u200D\uFEFF↵]/g, "")));
```

Comment:
This pre-existing bit of regex deals with empty values like covered in this test - I don't think we want to lose this functionality !
```ts
test("ignores empty values", () => {
  const [graph, ops] = add({
    id: "test",
    type: 100,
    data: {
      info: "\u200B​",
      policyRef: "​\n",
      empty: "​",
      another: "↵",
      howMeasured: undefined,
      text: "efef",
      description: " ​",
    },
  })({});
  expect(graph).toEqual({
    _root: {
      edges: ["test"],
    },
    test: {
      type: 100,
      data: {
        text: "efef",
      },
    },
  });
  expect(ops).toEqual([
    { oi: { edges: ["test"] }, p: ["_root"] },
    { oi: { data: { text: "efef" }, type: 100 }, p: ["test"] },
  ]);
});
```